src/main/scala/tile/RocketTile.scala

```diff
@@ -80,6 +81,111 @@ class RocketTile(
       val Description(name, mapping) = super.describe(resources)
       Description(name, mapping ++ cpuProperties ++ nextLevelCacheProperty ++ tileProperties ++ dtimProperty ++ itimProperty)
     }
+
+    override def getOMComponents(resourceBindingsMap: ResourceBindingsMap): Seq[OMComponent] = {
+      val cores = getOMRocketCores(resourceBindingsMap)
+      val icache = getOMICacheFromBindings(resourceBindingsMap)
+      cores ++: icache
+    }
+    def getOMICacheFromBindings(resourceBindingsMap: ResourceBindingsMap): Seq[OMComponent] = {
+      frontend.icache.device.getOMComponents(resourceBindingsMap)
+    }
+    def getOMRocketCores(resourceBindingsMap: ResourceBindingsMap): Seq[OMRocketCore] = {
+      val coreParams = rocketParams.core
+
+      val fpu= coreParams.fpu.map{f => OMFPU(fLen = f.fLen)}
+
+      val perfMon = if (coreParams.haveBasicCounters || coreParams.nPerfCounters > 0) {
+        Some(OMPerformanceMonitor(
+          specifications = List[OMSpecification](PrivilegedArchitectureExtensions.specVersion(MachineLevelISA, "1.10")),
+          hasBasicCounters = coreParams.haveBasicCounters,
+          nAdditionalCounters = coreParams.nPerfCounters
+        ))
+      }
+      else { None }
+
+      val pmp = if (coreParams.pmpGranularity > 0 || coreParams.nPMPs > 0) {
+        Some(OMPMP(
+          specifications = List[OMSpecification](PrivilegedArchitectureExtensions.specVersion(MachineLevelISA, "1.10")),
+          nRegions = coreParams.nPMPs,
+          granularity = coreParams.pmpGranularity
+        ))
+      }
+      else { None }
+
+      val mulDiv = coreParams.mulDiv.map{ md => OMMulDiv.makeOMI(md, xLen)}
+
+      val baseInstructionSet = xLen match {
+        case 32 => if (XLen == 32) RV32E else RV32I // TODO coreParams.useRVE
```

Comment:
Why is this here? Rocket doesn't support RVE.

Reply:
comment removed